docs/options.md

```diff
@@ -211,6 +211,21 @@ Align colons in multiline object literals (not applied with any of the JSON pars
 | ------- | --------------------------- | ------------------------------- |
 | `false` | `--align-object-properties` | `alignObjectProperties: <bool>` |
 
+## Break before statement
+
+Puts a hard break before single, non-block statement following: `if`, `else`, `with`, `for`, `for await`, `for each`, `do` and `while`.
```

Comment:
```suggestion
Force a hard break before single, non-block statement following: `if`, `else`, `for`, `for await`, `for each`, `do` and `while`, and `with`.
```

rationale: "soft break" vs "hard break" is an internal distinction; I think "soft break" is not quite abstract enough for external documentation. I am proposing a low-impact way to avoid the use of "soft break" in the documentation. I would also propose moving `with` to the end as discussed in another comment.